app/components/DatePicker/index.js

```diff
@@ -39,6 +39,7 @@ export default class DatePicker extends Component {
     instructions: null,
     attachment: 'left',
     disabled: false,
+    onChange: f=>f
```

Comment:
Can you tell me why this is necessary? Reading the rest of the code I see the `childChanged` function, but it isn't clear to me whats going on.

Reply:
the childChanged function is being passed down from the Group component to track changes so that when the reorder happens and the child components are re-rendered the users changes remain intact. The default function is there because in a case when the Field component is used outside of the Group component it still calls the onChange function so it would error without a function declaration.

Reply:
Got it, that makes sense. So each Group component is a little store for the changes tracked through the Group Rows, and it can move em around as needed. Thanks for the explanation 💯